use std::error::Error;

/// ## 2.2.1 读取数据集
///
#[test]
pub fn reading_the_dataset() -> Result<(), Box<dyn Error>> {

    Ok(())
}
